development/Dockerfile

```diff
@@ -1,7 +1,9 @@
-ARG NAUTOBOT_VER="1.0.1"
-ARG PYTHON_VER=3.8
+ARG PYTHON_VER
```

Comment:
This was copied directly from your SSOT plugin https://github.com/nautobot/nautobot-plugin-ssot-ipfabric/blob/develop/development/Dockerfile

Reply:
All our projects are defined this way to default on these args.  It's not a big deal, but we should stay consistent. I believe I was troubleshooting some local errors with w/ builds on py3.10 and never put it back. I will update SSOT, thanks for pointing that out!

Reply:
But if using invoke the nautobot and python versions get overridden.  Why do I have to change the version in my invoke.yml and then go to Dockerfile and change it there?

Reply:
That's a fair and true assessment, as we do use invoke to provide build args. I was mostly pointing this out because we do provide a default as a standard in our projects. The SSOT was an exception due to local troubleshooting.